Raven.Voron/Voron/StorageEnvironment.cs

```diff
@@ -671,11 +671,17 @@ private Task FlushWritesToDataFileAsync()
                                 throw new VoronUnrecoverableErrorException("Error occurred during flushing journals to the data file",
                                     new Win32Exception(sehException.HResult));
                             }
+                            catch (Exception e)
+                            {
```

Comment:
Thanks for the review, I missed it when I went over the code.
Yes, this has to be fixed.